app/src/main/java/com/alexbar/layoutbasic/utils/MoviesData.kt

```diff
@@ -0,0 +1,63 @@
+package com.alexbar.layoutbasic.utils
+
+import android.content.Context
+import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+import com.google.gson.reflect.TypeToken
+import java.io.IOException
+
+fun getJsonDataFromAsset(
```

Comment:
regarding this function, is it better to define it as extension function,  this way is a nice way to use this kind of method , like :

```
fun Context.getJsonDataFromAsset(fileName: String): String? {
    val jsonString: String
    try {
        jsonString = assets.open(fileName).bufferedReader().use {
            it.readText()
        }
    } catch (exp: IOException) {
        exp.printStackTrace()
        return null
    }

    return jsonString
}
```

and you can use it like :

`val jsonString = context.getJsonDataFromAsset("file.json")
`